### PR TITLE
[DOCS-462] Updating style guide with list styles description

### PR DESCRIPTION
**Pull request by rajkaramchedu**
@jimmyjames please review.

### CODE REVIEW

File: contributing/style-guide.rst [hunk 1]
```diff
@@ -92,6 +92,81 @@ Use sentence case, as in “Subscription management,” and not the title case,
...
+
+If it is an incomplete sentence, do not end the list element with a period.
+
+**Example:** 
+
+    SmartThings platform supports various Hub scenarios such as: 
+
+    - There may not be a hub at all
+    - SmartApps may run across both cloud and Hub connected devices
+    - There may be multiple Hubs
```
Comment (jimmyjames) [inline contributing/style-guide.rst]: I think we need to use some simpler examples and/or be convinced these aren't complete sentences... for example, SmartApps may run across both cloud and Hub connected devices" is a complete sentence (has a subject, verb, composes a complete thought).

Comment (rajkaramchedu) [inline contributing/style-guide.rst]: :- ) Hmm, I guess I didn't have my thinking cap on. These are indeed complete sentences.

**Comment by rajkaramchedu** [comment]
@jimmyjames replaced examples for incomplete sentences.

**Comment by rajkaramchedu** [comment]
Obsolete, closing. Refer to #589

**Pull request closed without merging**